zipline/examples/__init__.py

```diff
@@ -0,0 +1,17 @@
+from glob import glob
+from importlib import import_module
+import os
+
+for f in os.listdir(os.path.dirname(__file__)):
+    if not f.endswith('.py') or f == '__init__.py':
+        continue
+    modname = f[:-len('.py')]
+    globals()[modname] = import_module('.' + modname, package=__name__)
```

Comment:
Sorry, I meant, why are we walking the file system looking for submodules to import and add as attributes of this module, when `import module.submodule` should do that already?